patchwork-extensions-bakedmodel/src/main/java/net/minecraftforge/client/extensions/IForgeBakedModel.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client.extensions;
+
+import java.util.List;
+import java.util.Random;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import net.minecraftforge.client.model.data.IModelData;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.client.render.model.BakedModel;
+import net.minecraft.client.render.model.BakedQuad;
+import net.minecraft.client.texture.Sprite;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.world.BlockRenderView;
+
+/**
+ * Forge makes every BakeModel including vanilla BakedModels extend this interface.
+ * The handlePerspective and doesHandlePerspectives callback will be supported in a separated package.
```

Comment:
handlePerspective callback requires a lot of math classes, which obviously we dont have. "extensions-bakedmodel" might not be a good place for them. Furthermore, the internal detail of the way forge handle perspectives changes a lot since 1.15, so I'm still thinking...It might not worth to implement the handlePerspective callback in 1.14.